ibis/backends/pyspark/tests/conftest.py

```diff
@@ -198,9 +198,8 @@ def connect(data_directory):
 
 
 @pytest.fixture(scope='session')
-def client():
-    session = SparkSession.builder.getOrCreate()
-    client = ibis.backends.pyspark.Backend().connect(session)
+def client(data_directory):
+    client = get_pyspark_testing_client(data_directory)
```

Comment:
This is another fix that is necessary to get pyspark tests passing. It's not clear to me whether the current CI is exercising this code path.

Reply:
Moving to another patch

Reply:
#2926